README.md

```diff
@@ -251,3 +267,17 @@ The first argument passed to `create_collection` determines which type of collec
 3. calendar
 4. timeline
 5. gallery
+
+## Troubleshooting
+### No results returned when attempting to get a page
+Additionnaly to your `token_v2` credentials, you'll need to grab the `x-notion-active-user-header` from a Notion AJAX API call.
```

Comment:
Hey! Good catch on adding a troubleshooting section. Could we change this around to say:

"If an empty hash is returned when you attempt to retrieve a Notion page, you'll need to include the `x-notion-active-user-header` when instantiating the Notion Client. The endpoint ...**[rest can stay the same]**"